crates/proc-macro-srv/src/tests/mod.rs

```diff
@@ -136,23 +238,41 @@ fn test_fn_like_mk_literals() {
         "fn_like_mk_literals",
         r#""#,
         expect![[r#"
+            SUBTREE $$ 1 1
+
+
+
             SUBTREE $$ 1 1
               LITERAL ByteStr byte_string 1
               LITERAL Char c 1
               LITERAL Str string 1
+              LITERAL CStr cstring 1
               LITERAL Float 3.14f64 1
+              LITERAL Float -3.14f64 1
               LITERAL Float 3.14 1
+              LITERAL Float -3.14 1
               LITERAL Integer 123i64 1
-              LITERAL Integer 123 1"#]],
+              LITERAL Integer -123i64 1
+              LITERAL Integer 123 1
+              LITERAL Integer -123 1"#]],
         expect![[r#"
+            SUBTREE $$ 42:2@0..100#ROOT2024 42:2@0..100#ROOT2024
+
+
+
             SUBTREE $$ 42:2@0..100#ROOT2024 42:2@0..100#ROOT2024
               LITERAL ByteStr byte_string 42:2@0..100#ROOT2024
               LITERAL Char c 42:2@0..100#ROOT2024
               LITERAL Str string 42:2@0..100#ROOT2024
+              LITERAL CStr cstring 42:2@0..100#ROOT2024
               LITERAL Float 3.14f64 42:2@0..100#ROOT2024
+              LITERAL Float -3.14f64 42:2@0..100#ROOT2024
               LITERAL Float 3.14 42:2@0..100#ROOT2024
+              LITERAL Float -3.14 42:2@0..100#ROOT2024
               LITERAL Integer 123i64 42:2@0..100#ROOT2024
-              LITERAL Integer 123 42:2@0..100#ROOT2024"#]],
+              LITERAL Integer -123i64 42:2@0..100#ROOT2024
+              LITERAL Integer 123 42:2@0..100#ROOT2024
+              LITERAL Integer -123 42:2@0..100#ROOT2024"#]],
     );
```

Comment:
This output is bugged, the `-` is joined to the literal which is incorrect in our token tree model